src/SDK.ts

```diff
@@ -1014,12 +1036,14 @@ export default class SDK implements ISDK {
 
     return new Promise(async (resolve, reject) => {
       try {
+        const backendTimer = Timer.TIMER();
```

Comment:
shall we move it outside of try, so it can be destroyed in catch in case of error?, we should add the backennd elapsed time in the error telemetry as well

Reply:
Yes sir! Done, now the backendTimer is created outside to the try/catch scope and used in the catch as well, so we will know how long it took for the calls to fail, in addition to how long it took them to succeed.